spec.bs

```diff
@@ -1973,10 +1993,10 @@ To <dfn>fill in the contribution</dfn> given a
 following steps. They return a {{PAHistogramContribution}}.
 1. Let |bucket| be |contribution|["{{PAExtendedHistogramContribution/bucket}}"].
 1. If |bucket| is a {{PASignalValue}}, set |bucket| to the result of [=filling
-    in the signal value=] given |bucket|, 65535 and |leadingBidInfo|.
+    in the signal value=] given |bucket|, 2<sup>128</sup>−1 and |leadingBidInfo|.
 1. Let |value| be |contribution|["{{PAExtendedHistogramContribution/value}}"].
 1. If |value| is a {{PASignalValue}}, set |value| to the result of [=filling in
-    the signal value=] given |value|, 2<sup>128</sup>−1 and |leadingBidInfo|.
+    the signal value=] given |value|, 65535 and |leadingBidInfo|.
```

Comment:
Can the value be 65536?

Reply:
Ah good catch -- turns out the code actually enforces 2^31-1. (And the budget checking will just fail if a high value is used.)